.github/CODEOWNERS

```diff
@@ -23,7 +23,7 @@
 /hedera-node/hedera-mono-service/               @hashgraph/hedera-services
 /hedera-node/hapi*/                             @hashgraph/hedera-services
 /hedera-node/hedera-admin*/                     @hashgraph/hedera-services @kimbor @povolev15
-/hedera-node/hedera-app*/                       @hashgraph/hedera-services @hashgraph/platform-base
+/hedera-node/hedera-app*/                       @hashgraph/hedera-services @rbair23 @jasperpotts @netopyr
```

Comment:
```suggestion
/hedera-node/hedera-app*/                       @hashgraph/hedera-services @hashgraph/hedera-base @rbair23 @jasperpotts 
```

Reply:
I would like to explicitly leave it as @netopyr and not all of hedera-base.